test/JDBC/src/main/java/com/sqlsamples/batch_run.java

```diff
@@ -9,7 +9,6 @@
 import static com.sqlsamples.Statistics.exec_times;
 import static com.sqlsamples.Statistics.curr_exec_time;
 import static com.sqlsamples.Statistics.sla;
-import static com.sqlsamples.Config.checkParallelQueryExpected;
```

Comment:
why are we removing this ?

Reply:
We are already importing everything using import static com.sqlsamples.Config.*, hence redundant